aspnetcore/fundamentals/configuration/index.md

```diff
@@ -23,23 +23,62 @@ App configuration in ASP.NET Core is based on key-value pairs established by *co
+
+::: moniker-end
+
+Code examples that follow and in the sample app use the <xref:Microsoft.Extensions.Configuration> namespace:
 
 ```csharp
 using Microsoft.Extensions.Configuration;
 ```
 
-The *options pattern* is an extension of the configuration concepts described in this topic. Options uses classes to represent groups of related settings. For more information on using the options pattern, see <xref:fundamentals/configuration/options>.
+The *options pattern* is an extension of the configuration concepts described in this topic. Options use classes to represent groups of related settings. For more information, see <xref:fundamentals/configuration/options>.
 
 [View or download sample code](https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/fundamentals/configuration/index/samples) ([how to download](xref:index#how-to-download-a-sample))
 
 ## Host versus app configuration
 
-Before the app is configured and started, a *host* is configured and launched. The host is responsible for app startup and lifetime management. Both the app and the host are configured using the configuration providers described in this topic. Host configuration key-value pairs become part of the app's global configuration. For more information on how the configuration providers are used when the host is built and how configuration sources affect host configuration, see [The host](xref:fundamentals/index#host).
+Before the app is configured and started, a *host* is configured and launched. The host is responsible for app startup and lifetime management. Both the app and the host are configured using the configuration providers described in this topic. Host configuration key-value pairs are also included in the app's configuration. For more information on how the configuration providers are used when the host is built and how configuration sources affect host configuration, see <xref:fundamentals/index#host>.
 
 ## Default configuration
 
-Web apps based on the ASP.NET Core [dotnet new](/dotnet/core/tools/dotnet-new) templates call <xref:Microsoft.AspNetCore.WebHost.CreateDefaultBuilder*> when building a host. <xref:Microsoft.AspNetCore.WebHost.CreateDefaultBuilder*> provides default configuration for the app in the following order:
+::: moniker range=">= aspnetcore-3.0"
+
+Web apps based on the ASP.NET Core [dotnet new](/dotnet/core/tools/dotnet-new) templates call <xref:Microsoft.Extensions.Hosting.Host.CreateDefaultBuilder*> when building a host. `CreateDefaultBuilder` provides default configuration for the app in the following order:
+
+The following applies to apps using the [Generic Host](xref:fundamentals/host/generic-host). For details on the default configuration when using the [Web Host](xref:fundamentals/host/web-host), see the [ASP.NET Core 2.2 version of this topic](xref:fundamentals/configuration/index?view=aspnetcore-2.2).
```

Comment:
The QS `view=aspnetcore-2.2` in `[ASP.NET Core 2.2 version of this topic](xref:fundamentals/configuration/index?view=aspnetcore-2.2)` is ignored. Perhaps we need an include with a screenshot showing how to select the version. cc @tdykstra @scottaddie

Reply:
Why not use a non-xref link in that case if xreef links don't work with version qs?

Reply:
Lets go with @tdykstra  suggestion.

Reply:
Sounds good to me. :rocket: